pkgs/development/node-packages/node-env.nix

```diff
@@ -372,37 +372,8 @@ let
         fi
     '';
 
-  # Derivations built with `buildNodePackage` can already be overriden with `override`, `overrideAttrs`, and `overrideDerivation`.
-  # This function introduces `overrideNodeAttrs` and it overrides the call to `buildNodePackage`.
-  #
-  # THIS FUNCTION IS TEMPORARY until we have a better mechanism in place:
-  # https://github.com/NixOS/nixpkgs/pull/96509#issuecomment-682381592
-  # YOU SHOULD NOT USE IT UNLESS YOU ACCEPT THAT.
-  makeOverridableNodePackage = f: origArgs:
-    let
-      ff = f origArgs;
-      overrideWith = newArgs: origArgs // (
-        let args = if stdenv.lib.isFunction newArgs then newArgs origArgs else newArgs;
-        in
-          assert stdenv.lib.assertMsg (args.__acceptOverrideNodeAttrsCanBeDroppedAnytime or false) ''
-            overrideNodeAttrs is temporary function that will be removed once a better mechanism exists.
-            Pass it `__acceptOverrideNodeAttrsCanBeDroppedAnytime = true;` to aknowledge the fact.
-          '';
-          builtins.removeAttrs args [ "__acceptOverrideNodeAttrsCanBeDroppedAnytime" ]
-      );
-    in
-      if builtins.isAttrs ff then (ff // {
-        overrideNodeAttrs = newArgs: makeOverridableNodePackage f (overrideWith newArgs);
-      })
-      else if builtins.isFunction ff then {
-        overrideNodeAttrs = newArgs: makeOverridableNodePackage f (overrideWith newArgs);
-        __functor = self: ff;
-      }
-      else ff;
-
-
```

Comment:
I'm not sure what's up with `makeOverridableNodePackage`. This was deleted by `generate.sh`, but I'm not sure if it should have been. It was implemented very recently.

@jtojnar is this deletion ok? Should `generate.sh` be preserving this function?

Reply:
It should not be deleted, it is required for `expo-cli`. But it seems to be possible to do without: https://github.com/NixOS/nixpkgs/pull/100015

Reply:
Thanks. I just rebased this PR with #100015 included, so this diff is now gone.